ui/framework/src/ui-framework/content/SavedView.ts

```diff
@@ -76,30 +67,8 @@ export class SavedView {
 
   /** Create props for a ViewState */
   public static viewStateToProps(viewState: ViewState): SavedViewProps {
-    const savedViewProps: SavedViewProps = {
-      bisBaseClass: ViewUtilities.getBisBaseClass(viewState.classFullName),
-      viewDefinitionProps: viewState.toJSON(),
-      categorySelectorProps: viewState.categorySelector.toJSON(),
-      displayStyleProps: viewState.displayStyle.toJSON(),
-    };
-
-    if (ViewUtilities.isSpatial(savedViewProps.bisBaseClass)) {
-      const spatialViewState = viewState as SpatialViewState;
-      savedViewProps.modelSelectorProps = spatialViewState.modelSelector.toJSON();
-    }
-
-    if (ViewUtilities.isSheet(savedViewProps.bisBaseClass)) {
-      const sheetViewState = viewState as SheetViewState;
-      // TODO - Create valid SheetProps from SheetViewState
-      savedViewProps.sheetProps = {
-        width: sheetViewState.sheetSize.x, height: sheetViewState.sheetSize.y,
-        classFullName: "BisCore:Sheet",
-        code: { spec: "", scope: "" },
-        model: "",
-      };
-      savedViewProps.sheetAttachments = sheetViewState.attachmentIds;
-    }
-
+    const savedViewProps = viewState.toProps() as SavedViewProps;

```

Comment:
@bsteinbk @DanEastBentley A new property was added to ViewStateProps. I added ViewState.toProps() and modified the SavedView code to use it instead of manually producing JSON, so that this and future added properties will not be omitted by saved views.